integration-tests/run-scenario.sh

```diff
@@ -14,8 +14,94 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e
-set -o xtrace
+#set -e
+#set -o xtrace
+
+DIR=$2
+FILE1=${DIR}/infrastructure.properties
+FILE2=${DIR}/testplan-props.properties
+
+
+PROP_OS=OS                       #OS name e.g. centos
+PROP_TEST_MODE=TEST_MODE
+PROP_DB_URL=DatabaseHost
+PROP_DB_TYPE=DBEngine
+PROP_DB_UN=db_username
+PROP_DB_PW=db_password
+
+os=`cat ${FILE2} | grep -w "$PROP_OS" ${FILE1} ${FILE2}| cut -d'=' -f2`
+test_mode=`cat ${FILE2} | grep -w "$PROP_TEST_MODE" ${FILE1} ${FILE2}| cut -d'=' -f2`
+db_url=`cat ${FILE2} | grep -w "$PROP_DB_URL" ${FILE1} | cut -d'=' -f2`
+db_type=`cat ${FILE2} | grep -w "$PROP_DB_TYPE" ${FILE1} ${FILE2} | cut -d'=' -f2`
+db_username=`cat ${FILE2} | grep -w "$PROP_DB_UN" ${FILE1} ${FILE2} | cut -d'=' -f2`
+db_password=`cat ${FILE2} | grep -w "$PROP_DB_PW" ${FILE1} ${FILE2} | cut -d'=' -f2`
+
+echo "Db type is $db_type"
+echo "Db_url is $db_url"
+echo "test mode is $test_mode"
+echo "os is $os"
+echo "username is $db_username"
+echo "password is $db_password"
+
+
+if [ "${test_mode}" = "RELEASE" ]
+then
+  git clone https://github.com/wso2/product-sp.git
+  cd ${DIR}/product-sp/
+  value=$(git for-each-ref --sort=taggerdate --format '%(refname) %(taggerdate)' refs/tags | tail -1 | cut -d "/" -f 3 | cut -d " " -f 1)
+  git checkout tags/$value
+elif  [ "${test_mode}" = "SNAPSHOT" ]
+then
+  git clone https://github.com/wso2/product-sp.git
```

Comment:
Shall we specify the clone location relative to the DIR  path, just to be extra sure the clone happens at the right location